API/Program.cs

```diff
@@ -158,7 +158,7 @@
     {
         context.HttpContext.Response.StatusCode = StatusCodes.Status429TooManyRequests;
         const string readMore =
-            "Read more about our rate limits at https://github.com/osu-tournament-rating/otr-wiki/blob/master/api/limits/en.md";
+            "Read more about our rate limits at https://github.com/osu-tournament-rating"; // Stage will change this when there is an actual link for this./
```

Comment:
```suggestion
            // TODO: 
            "Read more about our rate limits at https://github.com/osu-tournament-rating";
```
Let's try to avoid inline comments like this as it doesn't fit the conventions of the rest of the solution. For situations like this I'd suggest using a `// TODO: ` comment. However see my review comment for a better approach.

Reply:
Also, if you were going to leave a placeholder link, it would make more sense for it to link to the API docs instead of the repo.